db/entities/attempt.go

```diff
@@ -5,22 +5,44 @@ import (
 	"encoding/json"
 )
 
+type Attempt struct {
+	ID            string            `json:"id" db:"id"`
```

Comment:
The primary key of the database preferably uses an auto-incrementing numerical ID. This is more friendly for new insertions into the database.

Reply:
To distinguish each different Attempt request, a new ID field can be set.

Reply:
Delivery attempts of an event to an endpoint can be distinguished by the AttemptNumber, which is an auto-incrementing numerical ID.